requirements.txt

```diff
@@ -5,9 +5,9 @@ inflection==0.2.1
 unicodecsv==0.14.1
 werkzeug==0.10.4
 
-git+https://github.com/alphagov/digitalmarketplace-utils.git@21.2.3#egg=digitalmarketplace-utils==21.2.3
+git+https://github.com/alphagov/digitalmarketplace-utils.git@21.4.0#egg=digitalmarketplace-utils==21.4.0
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@1.1.0#egg=digitalmarketplace-content-loader==1.1.0
```

Comment:
Would be good to also bump the content loader to 1.2.0. This is because the frameworks repo introduced a breaking change for v2.0.0 and requires the latest version of the content loader (1.2.0) to support this. (I think nothing would actually break as the buyer app doesn't call that new functionality but no harm being preemptive)